### PR TITLE
Update Event Schedule

### LINKED ISSUE #320: [Feature] Update Event Schedule Section
[Designs](https://www.figma.com/file/NzIlQyV6bXIksv9SyIFjj3/Live-Site-Wireframes?type=design&node-id=158-743&mode=design&t=7GS8hzqiKkaMniJI-0)

AC:

- [x] Update event schedule section to reflect design
- [x] Ensure desktop responsiveness
- [x] Light and dark mode

Image

@mike10911 @Aaryan1203

### PR DESCRIPTION
**Pull request by Mandy-cyber**
Updated the event schedule section to look like the one in the [Figma designs](https://www.figma.com/file/NzIlQyV6bXIksv9SyIFjj3/Live-Site-Wireframes?type=design&node-id=158-743&mode=design&t=7GS8hzqiKkaMniJI-0)
Fixes #320 

Changelist:

- Changed description section font size and colors
- Changed background colors and text colors of the actual calendar and tabs
- Changed the colors and positioning of buttons
- Changed arrow color
- Changed size of calendar so it's inline with the tabs on the side

Screenshots & Screencasts:
[PR.webm](https://github.com/HackBeanpot/mono-repo/assets/67931161/2b1accbb-38f5-45b2-bccb-efb14b3113df)

### CODE REVIEW

File: main-site/components/events-calendar-section/texts/DesktopTexts.tsx [hunk 1]
```diff
@@ -40,15 +40,23 @@ const DesktopTexts: React.FC = ({isDay}) => {
               {event.description && <EventsP>{event.description}</EventsP>}
             </EventsLeftContainer>
             <EventsRightContainer>
+<<<<<<< HEAD
+              <EventsImage src={ event.image || MissingPhoto}></EventsImage>
+=======
               {event.image &&
               <EventsImage src={ event.image}></EventsImage>
               }
+>>>>>>> 68dd83ee2593b77f98d857147e44094f55659acc
```
Comment (Aaryan1203) [inline main-site/components/events-calendar-section/texts/DesktopTexts.tsx]: Whats this for?

File: main-site/components/events-calendar-section/texts/DesktopTexts.tsx [hunk 2]
```diff
@@ -40,15 +40,23 @@ const DesktopTexts: React.FC = ({isDay}) => {
               {event.description && <EventsP>{event.description}</EventsP>}
             </EventsLeftContainer>
             <EventsRightContainer>
+<<<<<<< HEAD
+              <EventsImage src={ event.image || MissingPhoto}></EventsImage>
+=======
```
Comment (Aaryan1203) [inline main-site/components/events-calendar-section/texts/DesktopTexts.tsx]: I think you should get rid of the merge conflict text here

File: main-site/components/sponsors-section/SponsorsSection.styles.ts [hunk 3]
```diff
@@ -19,6 +19,10 @@ const StyledSponsorsHeader = styled(H2)`
 `;
 
 const StyledContactText = styled(H4)`
+<<<<<<< HEAD
+  color: ${colors.TEXT_BROWN} !important;
+=======
+>>>>>>> 68dd83ee2593b77f98d857147e44094f55659acc
```
Comment (Aaryan1203) [inline main-site/components/sponsors-section/SponsorsSection.styles.ts]: same thing here and i think u left them there for a couple files

File: main-site/yarn.lock [hunk 4]
```diff
@@ -2489,6 +2489,11 @@
   dependencies:
     "@types/node" "*"
 
+"@types/stylis@^4.0.2":
+  version "4.2.4"
+  resolved "https://registry.npmjs.org/@types/stylis/-/stylis-4.2.4.tgz"
+  integrity sha512-36ZrGJ8fgtBr6nwNnuJ9jXIj+bn/pF6UoqmrQT7+Y99+tFFeHHsoR54+194dHdyhPjgbeoNz3Qru0oRt0l6ASQ==
+
```
Comment (Aaryan1203) [inline main-site/yarn.lock:2496]: Also not sure if we should be pushing up yarn lock files i forget when were allowed to

**Review by Aaryan1203 — CHANGES_REQUESTED** [review]
Just left a few comments and I also noticed when in mobile view i think the formatting is a little off. Maybe you can add some padding and center it. Other than that everything looks good!
image

File: main-site/components/events-calendar-section/texts/DesktopTexts.tsx (hunk 1 above)
Comment (Mandy-cyber) [reply main-site/components/events-calendar-section/texts/DesktopTexts.tsx]: omg i thought all the merge conflicts were fixed, my bad. fixing now

File: main-site/yarn.lock (hunk 4 above)
Comment (Mandy-cyber) [reply main-site/yarn.lock:2496]: oh idk, if so my bad

**Review by Aaryan1203 — CHANGES_REQUESTED** [review]
Perfect, it looks better now. I think it still looks a little off in mobile view but after that looks good for me!

**Review by mike10911 — APPROVED** [review]
LGTM. FUCK THOSE TAGS

**Review by Aaryan1203 — APPROVED** [review]
LGTM!! great job Mandy

**Pull request merged**